docs/core/tools/dotnet-nuget-push.md

```diff
@@ -68,7 +68,7 @@ The command pushes an existing package. It doesn't create a package. To create a
 
 - **`-s|--source <SOURCE>`**
 
-  Specifies the server URL. This option is required unless `DefaultPushSource` config value is set in the NuGet config file.
+  Specifies the server URL. NuGet identifies a UNC or local folder source and simply copies the file there instead of pushing it using HTTP. Also, starting with NuGet 3.4.2, this is a mandatory parameter unless the NuGet config file specifies a `DefaultPushSource` value. For more information, see [Configuring NuGet behavior](/nuget/consume-packages/configuring-nuget-behavior).
```

Comment:
Would it make sense to help it stand out to wrap the note about NuGet version >= 3.4.2 in an important block quote?